doc/source/setup-helm.rst

```diff
@@ -9,6 +9,12 @@ Helm packages are called *charts*.
 We will be installing and managing JupyterHub on
 our Kubernetes cluster using a Helm chart.
 
+Charts are abstractions describing how to install packages onto a Kubernetes
+cluster. When a chart is deployed, it works as a templating engine to populate
+multiple `yaml` files for package dependencies with the required variables, and
```

Comment:
```suggestion
multiple ``yaml`` files for package dependencies with the required variables, and
```

I *think* we need double backticks everywhere because restructured text. Double the fun of Markdown :)

Reply:
Thanks, new thing learned! I followed the style of the rest of the document, but perhaps that is something that needs updating throughout. Though whichever engine is used to build the Read the Docs webpage seems to handle single backticks for inline Markdown fine?

Reply:
Ah, well, maybe there is some magic extension that we use or so. Or just me not understanding how reStructured text really works. (Can you tell I am basically at war with rST? :)

If the preview/local rendering works (`make html` in the docs directory probably) then I will retract my comment.

Reply:
Yep, single backticks render fine :)